qemu/tests/vsock_test.py

```diff
@@ -95,6 +94,50 @@ def nc_vsock_connect(nc_vsock_bin, guest_cid, port):
         output_params=("vsock_%s_%s" % (guest_cid, port),))
 
 
+def send_data_from_guest_to_host(guest_session, nc_vsock_bin,
+                                 guest_cid, tmp_file, file_size=1000):
+    """
+    Generate a temp file and transfer it from guest to host via vsock
+
+    :param guest_session: Guest session object
+    :param nc_vsock_bin: Path to nc-vsock binary
+    :param guest_cid: Guest cid to connected
+    :param file_size: Desired file size to be transferred
+    :return: The host nc-vsock connection process
+    """
+
+    cmd_generate = 'dd if=/dev/urandom of=%s count=%s bs=1M' % (tmp_file, file_size)
+    guest_session.cmd_status(cmd_generate, timeout=600)
+    port = random.randrange(1, 6000)
```

Comment:
Could we reuse the port in L178 ?  But, seems it is not very important, a random value is ok ?

Reply:
@yanan-fu It should be independent from the test(function `run`), otherwise it can't be reused by other test case. Per @xiangchunfu a random value is okay.